common/cloudant_base_test.go

```diff
@@ -167,4 +167,18 @@ var _ = Describe(`Cloudant custom base service UT`, func() {
 		Expect(authenticator).ToNot(BeNil())
 		Expect(authenticator.AuthenticationType()).To(Equal(core.AUTHTYPE_IAM))
 	})
+
+	It("Validates URL trailing slash is stripped", func() {
+		cloudant, err := NewBaseService(&core.ServiceOptions{
+			URL:           "https://cloudant.example/",
+			Authenticator: &core.NoAuthAuthenticator{},
+		})
+		Expect(cloudant).ToNot(BeNil())
+		Expect(err).To(BeNil())
+
+		builder := core.NewRequestBuilder(core.GET)
+		_, err = builder.ResolveRequestURL(cloudant.Options.URL, "/db", map[string]string{})
```

Comment:
Optional:
What do you think about these solutions?
```suggestion
		_, err = builder.ResolveRequestURL(cloudant.Options.URL, "/db", nil)
```
OR
A path parameter check might be added.

Reply:
Using `nil` is fine, we are not using builder for anything real, so no risk of runtime exception.

But I'm not sure what additional benefits path parameter check would give us, we already confirmed we have valid `url.URL`, because there is no error and `go-sdk-core` [validates URL for us](https://github.com/IBM/go-sdk-core/blob/main/v5/core/request_builder.go#L186-L189)